src/styles/AddBook.css

```diff
@@ -0,0 +1,52 @@
+.bookForm {
+  padding: 1rem 3rem;
+}
+
+.formTitle {
+  margin: 0;
+  padding: 0.5rem 1rem;
+  color: #888;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  border-top: 1px solid rgb(222, 222, 222);
+  border-left: 1px solid rgb(222, 222, 222);
+  border-right: 1px solid rgb(222, 222, 222);
+  font-family: Montserrat, sans-serif;
+}
```

Comment:
- [ ] Good job styling this, but please kindly ensure that is spacing below the form as specified in the design.

### Your Design
![image](https://github.com/Sparktan24/Bookstore/assets/43995830/e822d4ac-9edd-4c19-9628-5d8e86a8e318)

### Original Design
![image](https://github.com/Sparktan24/Bookstore/assets/43995830/9b8f456a-5de2-4477-b733-38b42189416d)